contracts/proxy/ERC1967/ERC1967Upgrade.sol

```diff
@@ -82,28 +83,14 @@ abstract contract ERC1967Upgrade {
         bytes memory data,
         bool forceCall
     ) internal {
-        address oldImplementation = _getImplementation();
-
-        // Initial upgrade and setup call
-        _setImplementation(newImplementation);
-        if (data.length > 0 || forceCall) {
-            Address.functionDelegateCall(newImplementation, data);
-        }
-
-        // Perform rollback test if not already in progress
-        StorageSlot.BooleanSlot storage rollbackTesting = StorageSlot.getBooleanSlot(_ROLLBACK_SLOT);
-        if (!rollbackTesting.value) {
-            // Trigger rollback using upgradeTo from the new implementation
-            rollbackTesting.value = true;
-            Address.functionDelegateCall(
-                newImplementation,
-                abi.encodeWithSignature("upgradeTo(address)", oldImplementation)
+        if (StorageSlot.getBooleanSlot(_ROLLBACK_SLOT).value) {
```

Comment:
Regardless of where have this code, I'm mentioning again that we should have a comment explaining that we do this for backwards compatibility.

Reply:
is it good like that ?